doc/modules/classes.rst

```diff
@@ -143,43 +143,58 @@ Classes
    covariance.oas
    covariance.graph_lasso
 
+.. _model_selection_ref:
```

Comment:
Ok so all the "ref" tags were added in 6709ed5fd99e59e0b0d26d05b116d1789b957a66, the usage later got removed, and we kept adding the tags?